WordPress/Classes/Utility/WPImmuTableRows.swift

```diff
@@ -141,7 +141,7 @@ struct DestructiveButtonRow: ImmuTableRow {
         cell.textLabel?.text = title
         cell.textLabel?.numberOfLines = 0
         cell.textLabel?.lineBreakMode = .byWordWrapping
-
+        cell.selectionStyle = .none
```

Comment:
Well if we're going to remain consistent with the rest of the settings-like screens making the row tappable makes the most sense.